framework/console/controllers/AssetController.php

```diff
@@ -326,6 +327,11 @@ protected function loadTargets($targets, $bundles)
     protected function buildTarget($target, $type, $bundles)
     {
         $tempFile = $target->basePath . '/' . strtr($target->$type, ['{hash}' => 'temp']);
+        if (!is_file($tempFile)) {
+            FileHelper::createDirectory(dirname($tempFile), 0755, true);
```

Comment:
Should be at least `$target->basePath`.
And perhaps the check should be performed against `$target->basePath` as well.